layouts/New/Blog.tsx

```diff
@@ -1,11 +1,14 @@
+import { RssIcon } from '@heroicons/react/24/solid';
 import { getTranslations } from 'next-intl/server';
 import type { FC } from 'react';
 
 import { getClientContext } from '@/client-context';
 import WithBlogCategories from '@/components/withBlogCategories';
 import WithFooter from '@/components/withFooter';
 import WithNavBar from '@/components/withNavBar';
+import { Link } from '@/navigation.mjs';
```

Comment:
We should use the Link component from components folder

Reply:
you mean ?

```suggestion
import Link from '@/components/Link';
```

Reply:
https://github.com/nodejs/nodejs.org/blob/main/components/Link.tsx